src/quo/components/bottom_sheet/view.cljs

```diff
@@ -1,23 +1,25 @@
 (ns quo.components.bottom-sheet.view
-  (:require [reagent.core :as reagent]
-            [quo.animated :as animated]
-            [quo.react-native :as rn]
-            [quo.react :as react]
-            [quo.platform :as platform]
-            [cljs-bean.core :as bean]
-            [quo.components.safe-area :as safe-area]
-            [quo.components.bottom-sheet.style :as styles]
-            [quo.gesture-handler :as gesture-handler]
-            [quo.design-system.colors :as colors]))
+  (:require
```

Comment:
I don't know if it's possible to not break the line after the `:require`, but I'd prefer if we could get rid of this unnecessary line break. But if it's too much effort it's fine, we could improve this rule later.